schema-guide.md

```diff
@@ -1912,7 +1912,7 @@ authors:
 
 ### `definitions.orcid`
 
-- **type**: `uri` with pattern [`https://orcid\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}`](https://regex101.com/library/wvvVYE)
+- **type**: `uri` with pattern [`https://orcid\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}`](https://regex101.com/library/wvvVYE). Make sure to avoid any leading or trailing whitespace.
```

Comment:
```suggestion
- **type**: `uri` with pattern [`^https://orcid\.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]{1}$`](https://regex101.com/r/GLGGDO/1).
```

Updating the pattern and linking to an updated regex should be enough documentation, without the whitespace comment?

Reply:
Good catch, thanks